### ISSUE TITLE
[libtorch] update to 1.13.1

### ISSUE THREAD

**Issue by hushanjushi** [microsoft/vcpkg#29886]
Library name: libtorch

New version number: 1.13.1

New stable version of libtorch is 1.13.1 [now](https://pytorch.org/blog/PyTorch-1.13-release/).

**Comment by bsergean** [comment]
Hi there, I tried to build this on macOS-arm and it didn't work.

```
Downloading vcpkg-macos...
Computing installation plan...
sleef is only supported on '!uwp & !(arm & windows) & !(arm64 & osx)', which does not match arm64-osx. This usually means that there are known build failures, or runtime problems, when building other platforms. To ignore this and attempt to build sleef anyway, rerun vcpkg with `--allow-unsupported`.
```

The problem is with the sleef library, for which vcpkg use a released version, but there seems to be fixed in the master branch (https://github.com/shibatch/sleef/commit/e7b4784bbb9be93491a6910776a0a1c50a93955b)

**Comment by jimwang118** [comment]
> The problem is with the sleef library, for which vcpkg use a released version, but there seems to be fixed in the master branch ([shibatch/sleef@e7b4784](https://github.com/shibatch/sleef/commit/e7b4784bbb9be93491a6910776a0a1c50a93955b))

You can try to use the commit ID instead of the version number, update to the latest `sleef`, and compile on mac-os to see if it works. Because this modification was submitted after its final official release.

**Comment by hushanjushi** [comment]
> > The problem is with the sleef library, for which vcpkg use a released version, but there seems to be fixed in the master branch ([shibatch/sleef@e7b4784](https://github.com/shibatch/sleef/commit/e7b4784bbb9be93491a6910776a0a1c50a93955b))
> 
> You can try to use the commit ID instead of the version number, update to the latest `sleef`, and compile on mac-os to see if it works. Because this modification was submitted after its final official release.

I think we may wait for some days for a while. Pytorch is get 2.0 release before Mar. 15.
